docs-src/0.7/src/essentials/basics/signals.md

```diff
@@ -202,15 +209,15 @@ rsx! {
 }
 ```
 
-Calls to `.read()` access the current reactive scope, adding this scope to the list of subscribers to the Signal. By default, components attach a re-render effect to all signals accessed from witin their reactive scope.
+Calls to `.read()` access the current reactive scope, adding this scope to the list of subscribers to the Signal with a side effect that runs when that signal is changed. For components, the logic causes the component to queue a re-render side effect.
 
 ![Component effects](/assets/07/component-effect.png)
 
 There are other uses of reactive scopes beyond component re-renders. Hooks like `use_effect`, `use_memo`, and `use_resource` all implement functionality by leveraging a reactive scope that exists *outside* the rendering lifecycle.
 
 ## Automatic Batching
 
-By default, all `.write()` calls are batched in one "step" of your app. Dioxus does not synchronously run side-effects when you call `.write()`. Instead, the runtime waits for all events to be handled *first* before determining what side-effects need to be run. This provides automatic batching of `.write()` calls which is important both for performance and consistency in the UI.
```

Comment:
This isn't true for some hooks like memos. We always call the callbacks attached to write immediately, but they generally queue things with async tasks like spawn. This is also getting into the weeds of how the scheduler works. Maybe this would fit better in an advanced guide?